modules/mono/mono_gd/support/android_support.cpp

```diff
@@ -414,9 +414,8 @@ GD_PINVOKE_EXPORT int32_t monodroid_get_system_property(const char *p_name, char
 
 	if (r_value) {
 		if (len >= 0) {
-			*r_value = (char *)malloc(len + 1);
-			if (!*r_value)
-				return -1;
+			*r_value = (char *)memalloc(len + 1);
```

Comment:
This is freed at line 629 and should be changed to `memfree()` - or kept as `malloc()` here if @neikeq thinks it's better.

Reply:
Nice catch :slightly_smiling_face:.  I couldn't find any call to this method, maybe VSCode is skipping files that are ignored due to #if guards, I'll have to see if I can fix this.

I agree to use `malloc` here again; honestly, I'm scared of the scope this PR has, so maybe we should just stick with `malloc` where there are worries it might break something. The objective is to prevent null de-references from happening anyways.

Reply:
I prefer to keep it as it was as I'm not 100% sure it's always freed with the `monodroid_free` we implement above. I prefer not to risk.

Reply:
Fixed